core/server/master/src/test/java/alluxio/master/meta/UpdateCheckTest.java

```diff
@@ -40,23 +42,25 @@ public void userAgentStringEmpty() throws Exception {
     Mockito.when(EnvironmentUtils.isEC2()).thenReturn(false);
     Mockito.when(EnvironmentUtils.isCFT(Mockito.anyString())).thenReturn(false);
     Mockito.when(EnvironmentUtils.isEMR(Mockito.anyString())).thenReturn(false);
+    PowerMockito.mockStatic(EC2MetadataUtils.class);
+  }
 
+  @Test
+  public void userAgentStringEmpty() throws Exception {
     String userAgentString = UpdateCheck.getUserAgentString("cluster1");
+    Mockito.when(EC2MetadataUtils.getUserData())
```

Comment:
can we move this to `before` as well so that each test only mocks the methods specific to that test

Reply:
I tried to move it to before, but then I cannot override
```
Mockito.when(EC2MetadataUtils.getUserData()).thenThrow(new SdkClientException("Unable to contact EC2 metadata service."));
```
with 
```
Mockito.when(EC2MetadataUtils.getUserData()).thenReturn("actual values");
```
It will still throw exceptions